gg.minion.Minion.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>gg.minion.Minion</id>
+  <name>Minion</name>
+  <developer_name>Good Game Mods, LLC</developer_name>
+  <summary>Premier Addon Management</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary</project_license>
+  <url type="homepage">https://minion.mmoui.com/</url>
```

Comment:
why does this differ from the app-id

Reply:
Do you mean the URL? Another valid url for it is `https://minion.gg/`, I can place it there instead.

Reply:
that's, just asking. sometime people fell that a random domain if fine as an app-id